src/module.rs

```diff
@@ -264,14 +263,24 @@ impl Module {
     }
 
     // See GH issue #6
-    fn write_bitcode_to_file(&self, file: &File, should_close: bool, unbuffered: bool) -> bool {
+    #[cfg(unix)]
+    pub fn write_bitcode_to_file(&self, file: &File, should_close: bool, unbuffered: bool) -> bool {
+        use std::os::unix::io::AsRawFd;
+        use llvm_sys::bit_writer::LLVMWriteBitcodeToFD;
+
         // REVIEW: as_raw_fd docs suggest it only works in *nix
         // Also, should_close should maybe be hardcoded to true?
         unsafe {
             LLVMWriteBitcodeToFD(self.module, file.as_raw_fd(), should_close as i32, unbuffered as i32) == 0
         }
     }
 
+    #[cfg(windows)]
+    #[allow(unused_variables)]
+    pub fn write_bitcode_to_file(&self, file: &File, should_close: bool, unbuffered: bool) -> bool {
```

Comment:
Sweet! Thanks! I hope we'll be able to get this working on windows eventually, and if not we'll probably just remove the method completely from windows.